modules/nw-how-nw-iface-selected.adoc

```diff
@@ -0,0 +1,78 @@
+// Applies to 4.6 and newer.
+
+:ign-config-version: 3.1.0
+ifeval::[{product-version} > 4.6]
+:ign-config-version: 3.2.0
+endif::[]
+
+[id="nw-how-nw-iface-selected_{context}"]
+= How the network interface is selected
+
+For deployments on bare metal or with virtual machines that have more than one network interface controller (NIC), the NIC that {product-title} uses for communication with the Kubernetes API server is determined by a script that runs as a systemd unit.
+The `nodeip-configuration.service` systemd unit runs when the node boots.
+The script iterates through the network interfaces on the node and the first network interface that is configured with a subnet than can host the IP address for the API server is selected for {product-title} communication.
+
+After the script determines the correct NIC, the script creates the `/etc/systemd/system/kubelet.service.d/20-nodenet.conf` file.
+The `20-nodenet.conf` file sets the `KUBELET_NODE_IP` environment variable to the IP address that was selected.
+
+When the kubelet service starts, it reads the value of the environment variable from the `20-nodenet.conf` file and sets the IP address as the value to the `--node-ip` kubelet command-line argument.
+As a result, the kubelet service uses the selected IP address as the node IP address.
+
+If hardware or networking is reconfigured after installation, it is possible that the script for the `nodeip-configuration.service` unit can select a different NIC after a reboot and prevent {product-title} communication.
+
+If {product-title} communication is disrupted because a different NIC is selected, one strategy for overriding the selection process is to set the correct IP address explicitly.
```

Comment:
How does the user know this is the problem?

Reply:
I suggested that customers review the INTERNAL-IP column from the output of `oc get nodes -o wide`.